lib/signs/utilities/messages.ex

```diff
@@ -40,24 +45,50 @@ defmodule Signs.Utilities.Messages do
             Tuple.to_list(sign.source_config),
             Tuple.to_list(predictions),
             Tuple.to_list(alert_status),
-            Tuple.to_list(scheduled),
+            Tuple.to_list(first_scheduled_departures),
+            Tuple.to_list(last_scheduled_departures),
+            Tuple.to_list(recent_departures),
             Tuple.to_list(service_status)
           ])
         else
-          [{sign.source_config, predictions, alert_status, scheduled, service_status}]
+          [
+            {sign.source_config, predictions, alert_status, first_scheduled_departures,
+             last_scheduled_departures, recent_departures, service_status}
+          ]
         end
-        |> Enum.map(fn {config, predictions, alert_status, scheduled, service_status} ->
+        |> Enum.map(fn {config, predictions, alert_status, first_scheduled_departures,
+                        last_scheduled_departures, recent_departures, service_status} ->
```

Comment:
Bit of a nitpick, but could we rename `recent_departures` to something like `most_recent_departure` or something to be a little clearer on what that value is?